src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -2108,6 +2109,17 @@ public void testNetworksConnectContainer() throws Exception {
     assertThat(network.containers().size(), equalTo(1));
     assertThat(network.containers().get(containerCreation.id()), notNullValue());
 
+    final ContainerInfo containerInfo = sut.inspectContainer(containerCreation.id());
+    assertThat(containerInfo.networkSettings().networks().size(), is(2));
+    final AttachedNetwork attachedNetwork =
+        containerInfo.networkSettings().networks().get(networkName);
+    assertThat(attachedNetwork, is(notNullValue()));
+    assertThat(attachedNetwork.endpointId(), is(notNullValue()));
+    assertThat(attachedNetwork.gateway(), is(notNullValue()));
+    assertThat(attachedNetwork.ipAddress(), is(notNullValue()));
+    assertThat(attachedNetwork.ipPrefixLen(), is(notNullValue()));
+    assertThat(attachedNetwork.macAddress(), is(notNullValue()));
```

Comment:
I think that result depends from machine setup (if ipv6 is enabled or not). On my machine `ipv6Gateway()` and `globalIPv6Address()` return void String and `globalIPv6PrefixLen()` returns 0.
I added a check that verifies that `ipv6Gateway()` and `globalIPv6Address()` return not null values and `globalIPv6PrefixLen()` >=0. Is it ok?